src/tcgwars/logic/impl/gen4/CallOfLegends.groovy

```diff
@@ -522,279 +383,79 @@ public enum CallOfLegends implements LogicCardInfo {
           move "Juggling", {
             text "10× damage. Flip 4 coins. This attack does 10 damage times the number of heads."
             energyCost P, C
-            attackRequirement {}
             onAttack {
-              damage 0
+              flip 4, {
+                damage 10
+              }
             }
           }
 
         };
       case PIDGEOT_30:
-        return evolution (this, from:"Pidgeotto", hp:HP120, type:COLORLESS, retreatCost:0) {
-          weakness L
-          resistance F, MINUS20
-          move "Headwind", {
-            text "20 damage. more."
-            energyCost C, C, C, C
-            attackRequirement {}
-            onAttack {
-              damage 0
-            }
-          }
-          move "Quick Attack", {
-            text "40+ damage. Flip a coin. If heads, this attack does 40 damage plus 30 more damage."
-            energyCost C, C, C
-            attackRequirement {}
-            onAttack {
-              damage 40
-              flip { damage 30 }
-            }
-          }
-
-        };
+        return copy(Triumphant.PIDGEOT_29, this);
       case SKARMORY_31:
-        return basic (this, hp:HP080, type:METAL, retreatCost:1) {
-          weakness R
-          resistance P, MINUS20
-          move "Steel Coat", {
-            text "Energy card and attach it to 1 of your Pokémon. Shuffle your deck afterward."
-            energyCost M, M
-            attackRequirement {}
-            onAttack {
-              damage 0
-            }
-          }
-          move "Razor Wing", {
-            text "40 damage. "
-            energyCost M, C, C
-            attackRequirement {}
-            onAttack {
-              damage 0
-            }
-          }
-
-        };
+        return copy(Undaunted.SKARMORY_21, this);
       case SLOWKING_32:
-        return evolution (this, from:"Slowpoke", hp:HP080, type:PSYCHIC, retreatCost:2) {
-          weakness P
-          pokePower "Second Sight", {
-            text "Once during your turn , you may look at the top 3 cards of either player’s deck and put them back on top of that player’s deck in any order. This power can’t be used if Slowking is affected by a Special Condition."
-            actionA {
-            }
-          }
-          move "Psyshock", {
-            text "30 damage. Flip a coin. If heads, the Defending Pokémon is now Paralyzed."
-            energyCost P, C
-            attackRequirement {}
-            onAttack {
-              damage 0
-            }
-          }
-
-        };
+        return copy(HeartgoldSoulsilver.SLOWKING_12, this);
       case SNORLAX_33:
         return basic (this, hp:HP100, type:COLORLESS, retreatCost:4) {
           weakness F
           move "Layabout", {
             text "Remove all damage counters from Snorlax. Snorlax can’t use Layabout during your next turn."
             energyCost C, C, C
-            attackRequirement {}
+            attackRequirement {self.numberOfDamageCounters}
```

Comment:
```suggestion
            attackRequirement {assert self.numberOfDamageCounters : "$self not damaged}
```
Need to use the assert statement. Nice to use an error string to the player so they know why it failed.